themes/README.md

```diff
@@ -0,0 +1,63 @@
+# Mumble Theme(s)
+
+This directory contains the theme(s) Mumble ships with by default.
+
+A theme consists of the following components:
+- Style definitions (e.g. used colors, border widths, etc.) that are defined as QSS files (Qt's version of CSS)
+- Icons
+- Icon-Mapping that map a given name (alias) that is used in the source code to a particular icon in the theme
+
+The heart of a theme is its corresponding `.qrc` file. This file contains the Icon-Mapping and also a mapping for the QSS files. It is responsible of
+bringing all required resources into the final binary and making these resources available under the name chosen in it.
+
+
+## Editing icons
+
+Editing icons is very straight forward. Most of our icons are vector graphics (SVGs) and that is what you should create when assembling a new icon.
+These files can be viewed and edited using the Open Source and cross-platform tool [Inkscape](https://inkscape.org/).
+
+By editing these SVG files with Inkscape, the files get bloated with a lot of unnecessary stuff though, so in order to clean them up (again), we use
+the [SVGOMG web tool](https://jakearchibald.github.io/svgomg/). Just upload the SVG to their page and accept the default values. That usually produces
+good results. Make sure to actually test that the icon still looks the same _inside Mumble_ after you have stripped it in this way.
+
+As already mentioned, every icon needs to be referenced in the corresponding `.qrc` file in order to find its way into the final application. In there
+you also have to create a unique name-alias for it. An entry in that file looks like this:
+```xml
+<file alias="nameToBeUsedInSourceCode">path/to/icon.svg</file>
+```
+
+In the source code you can then refer to your new icon as `skin:nameToBeUsedInSourceCode`.
```

Comment:
application source code (?)

I think that is not necessarily clear here, or ambiguous